product_docs/docs/pem/9/monitoring_performance/notifications.mdx

```diff
@@ -265,3 +265,26 @@ You can use the following command to confirm that Nagios is properly configured:
 After confirming that Nagios is configured correctly, restart the Nagios service:
 
 `/usr/local/nagios/bin/nagios -d /usr/local/nagios/etc/nagios.cfg`
+
+## Customizing email templates
+
+Postgres Enterprise Manager monitors your system for conditions that require user attention and sends email notifications. Use the **Email Templates** to customize the email subject and the payload that the server sends if the current values deviate from the threshold values specified by an alert.
```

Comment:
Are we being consistent with using the long name vs the short name? at least in individual topics?